ui/src/main/java/com/doordeck/sdk/common/models/JWTHeader.java

```diff
@@ -1,222 +1,63 @@
-        this.phone_number = telephone;
-        this.telephone_verified = telephone_verified;
-        this.email = email;
-        this.exp = exp;
-        this.sub = sub;
-        this.email_verified = email_verified;
-        this.session = session;
-        this.name = name;
-        this.given_name = given_name;
-        this.family_name = family_name;
-        this.middle_name = middle_name;
-        this.iss = iss;
-        this.aud = aud;
-        this.refresh = refresh;
-        this.iat = iat;
-        this.sid = sid;
-        this.locale = locale;
-        this.zoneinfo = zoneinfo;
-        this.auth_time = auth_time;
-        this.picture = picture;
-    }
-
-    public String getPhone_number() {
-        return phone_number;
-    }
-
-    public void setPhone_number(String phone_number) {
-        this.phone_number = phone_number;
-    }
-
-    public Boolean getTelephone_verified() {
-        return telephone_verified;
-    }
-
-    public void setTelephone_verified(Boolean telephone_verified) {
-        this.telephone_verified = telephone_verified;
-    }
-
-    public String getEmail() {
-        return email;
-    }
-
-    public void setEmail(String email) {
-        this.email = email;
-    }
-
-    public int getExp() {
-        return exp;
-    }
-
-    public void setExp(int exp) {
-        this.exp = exp;
-    }
-
-    public String getSub() {
-        return sub;
-    }
-
-    public void setSub(String sub) {
-        this.sub = sub;
-    }
-
-    public Boolean getEmail_verified() {
-        return email_verified;
-    }
-
-    public void setEmail_verified(Boolean email_verified) {
-        this.email_verified = email_verified;
-    }
-
-    public String getSession() {
-        return session;
-    }
-
-    public void setSession(String session) {
-        this.session = session;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public String getGiven_name() {
-        return given_name;
-    }
-
-    public void setGiven_name(String given_name) {
-        this.given_name = given_name;
-    }
-
-    public String getFamily_name() {
-        return family_name;
-    }
-
-    public void setFamily_name(String family_name) {
-        this.family_name = family_name;
-    }
-
-    public String getMiddle_name() {
-        return middle_name;
-    }
-
-    public void setMiddle_name(String middle_name) {
-        this.middle_name = middle_name;
-    }
-
-    public String getIss() {
-        return iss;
-    }
-
-    public void setIss(String iss) {
-        this.iss = iss;
-    }
-
-    public String getAud() {
-        return aud;
-    }
-
-    public void setAud(String aud) {
-        this.aud = aud;
-    }
+public interface JWTHeader {
 
-    public boolean isRefresh() {
-        return refresh;
-    }
+    String sub();
+    String iss();
+    String session();
+    Boolean refresh();
```

Comment:
You could consider making this an OptionalBoolean because it isn't normally present on any JWT's except our own refresh token